src/mvAppItem.cpp

```diff
@@ -4088,11 +4205,30 @@ DearPyGui::GetEntityParser(mvAppItemType type)
 
         args.push_back({ mvPyDataType::Double, "x" });
         args.push_back({ mvPyDataType::Double, "y" });
-        args.push_back({ mvPyDataType::Integer, "x_offset", mvArgType::KEYWORD_ARG });
-        args.push_back({ mvPyDataType::Integer, "y_offset", mvArgType::KEYWORD_ARG });
+        args.push_back({ mvPyDataType::Integer, "x_offset", mvArgType::DEPRECATED_KEYWORD_ARG, "0", "Old way to set x offset of the label. Use `offset` argument instead.", "See the new offset argument." });
+        args.push_back({ mvPyDataType::Integer, "y_offset", mvArgType::DEPRECATED_KEYWORD_ARG, "0", "Old way to set y offset of the label. Use `offset` argument instead.", "See the new offset argument." });
+        args.push_back({ mvPyDataType::FloatList, "offset", mvArgType::KEYWORD_ARG, "(0.0, 0.0)", "Offset of the label, in pixels, relative to the coordinates." });
         args.push_back({ mvPyDataType::Bool, "vertical", mvArgType::KEYWORD_ARG, "False" });
 
-        setup.about = "Adds a label series to a plot.";
+        setup.about = "Adds a label series to a plot. x and y can only have one elements each.";
+        setup.category = { "Plotting", "Containers", "Widgets" };
+        break;
+    }
+    case mvAppItemType::mvDigitalSeries:             
+    {
+        AddCommonArgs(args, (CommonParserArgs)(
+            MV_PARSER_ARG_ID |
+            MV_PARSER_ARG_PARENT |
+            MV_PARSER_ARG_BEFORE |
+            MV_PARSER_ARG_SOURCE |
+            MV_PARSER_ARG_SHOW)
+        );
+
+        args.push_back({ mvPyDataType::DoubleList, "x" });
+        args.push_back({ mvPyDataType::DoubleList, "y" });
+        args.push_back({ mvPyDataType::Integer, "offset", mvArgType::KEYWORD_ARG, "0" });
```

Comment:
`offset`: This argument tells ImPlot where in the data arrays to start picking up elements. Using `offset`, `count`, and `stride`, one can build different plot views from a single large chunk of data. However, it's best suited for the C users of ImPlot. In DPG, we always specify `count` as the length of data array, and `offset` doesn't really make much sense. Another good example is that all other `ImPlot::PlotXXXSeries` functions also have `offset` and `stride` arguments, but DPG never uses them.

I'd suggest that we delete this argument on `mvDigitalSeries`.